renderdoc/driver/vulkan/vk_acceleration_structure.h

```diff
@@ -43,8 +87,35 @@ class VulkanAccelerationStructureManager
     bool isTLAS;
   };
 
+  struct Allocation
+  {
+    VkDeviceMemory mem = VK_NULL_HANDLE;
+    VkDeviceSize size = 0;
+    VkBuffer buf = VK_NULL_HANDLE;
+  };
+
+  struct RecordAndOffset
+  {
+    VkResourceRecord *record = NULL;
+    VkDeviceAddress address = 0x0;
+    VkDeviceSize offset = 0;
+  };
+
   explicit VulkanAccelerationStructureManager(WrappedVulkan *driver);
 
+  // Allocates readback mem and injects commands into the command buffer so that the input buffers
+  // are copied.
+  RDResult CopyInputBuffers(VkCommandBuffer commandBuffer,
+                            const VkAccelerationStructureBuildGeometryInfoKHR &info,
+                            const VkAccelerationStructureBuildRangeInfoKHR *buildRange,
+                            CaptureState state);
```

Comment:
This state seems redundant here - we don't do this during replay, and we don't need to differentiate between background and active capturing. Doing a quick look through unless I'm missing something this value is (ultimately) not used.

Same applies below to `CopyAccelerationStructure`.